server/controllers/posts.js

```diff
@@ -31,15 +37,24 @@ module.exports.getPostsByStoryId = (req, res) => {
 };
 
 module.exports.getPostsByLandmarkId = (req, res) => {
-  models.Post.getPostsByLandmarkId(req.body)
+  models.Post.getPostsByLandmarkId(req.params.id)
     .then(results => {
       res.status(200).send(results);
     });
 };
 
-module.exports.getPostsByFollowing = (req, res) => {
-  models.Post.getPostsByFollowing(req.body)
+module.exports.getPostsByFollowings = (req, res) => {
+  models.Following.getAllFollowings(req.params.id)
```

Comment:
Is req.params.id the id of /api/posts/followings/:id?

Reply:
Sorry, that's actually the current user id